### DISCUSSION TITLE
Spotify Links?

### DISCUSSION THREAD

**Discussion by KFL-Exhibits** [CollectionBuilder/collectionbuilder.github.io#114] [Q&A]
Hey all, I'm working on an exhibit on a podcasting project here at my institution and I was wondering if anyone had tried spotify links in CollectionBuilder? Would those be valid external links for exhibit objects?
Thanks! -Larry

**Comment by evanwill** [comment]
@KFL-Exhibits you could handle it a couple ways--> 

- out of the box, you could use the "record" display_template and put the link to the spotify item in "object_location"--it will show up as the "Link to Object" button on the Item page. 
- out of the box, you could put the link to the spotify item as a metadata column, e.g. "spotify_link", and use the "external_link" option in config-metadata.csv to make it display as a clickable link in the metadata of your Item page.
- with customization, you could tweak the Item page to use the spotify link from your metadata to put a customize link, like a big button with "listen on spotify" or something.
- with customization, you could use your spotify links to create a [spotify embed](https://developer.spotify.com/documentation/embeds) on the item page. This is pretty straightforward, and makes it so the user wouldn't have to leave the item page to listen.

**Reply by KFL-Exhibits** [reply]
Stellar, ok thank you. I wasn't sure using the embed code would work, but if that's an option I might figure out how to do that. Much appreciated! - Larry
